clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -284,6 +326,34 @@ CheckForIncompatibleAttributes(Sema &S,
   }
 }
 
+
+template <typename FPGALoopAttrT> static void
+CheckForDuplicationFPGALoopAttribute(Sema &S,
+                                     const SmallVectorImpl<const Attr *>
+                                     &Attrs, SourceRange Range) {
+  const FPGALoopAttrT *LoopFPGAAttr = nullptr;
+
+  for (const auto *I : Attrs) {
```

Comment:
It seems this function is wrong.
Consider the case:
[[intelfpga::max_concurrency_loop(2)]] [[intelfpga::ivdep(2)]]
for() 
According to source code it will say that there is duplication.
Am I missing something?

Reply:
No, you are not. Fixed.